src/nginx/module.cc

```diff
@@ -1004,6 +1004,16 @@ ngx_int_t ngx_esp_init_module(ngx_cycle_t *cycle) {
   conf.module_type = NGX_HTTP_MODULE;
   conf.cmd_type = NGX_HTTP_MAIN_CONF;
 
+  // Fixed crash with v1.15.0
+  conf.conf_file =
+      (ngx_conf_file_t *)ngx_pcalloc(cycle->pool, sizeof(ngx_conf_file_t));
```

Comment:
Seems like the code is not formatted properly.

Reply:
Code was formatted using clang-format tool